pages/business/setup/signup.tsx

```diff
@@ -91,6 +95,10 @@ const SetupSignupPage: React.FC<WithOwnerProps> = () => {
             <Form>
               <Input name="name" label={isCareEnv ? 'Ihr Name' : 'Dein Name'} />
               <Box height={4} />
+              <Input name="companyName" label="Name Ihres Unternehmens" />
```

Comment:
```suggestion
              <Input name="companyName" label={isCareEnv ? 'Name Ihres Unternemen' : 'Name von deinem Unternehmen' />
```
Not sure if thats the best possible labels, but we should  keep it consistent with the formal/ informat style

Reply:
Hab ich lokal schon angepasst. Aber ich versuche mein eslint zu fixen :(

Reply:
Should automatically be fixable with `npm run lint -- --fix`